nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/test/groovy/org/apache/nifi/processors/script/ExecuteScriptGroovyTest.groovy

```diff
@@ -149,6 +150,7 @@ class ExecuteScriptGroovyTest extends BaseScriptTest {
         }
     }
 
+    @Ignore("This test fails intermittently when the serial execution happens faster than pooled")
```

Comment:
You are probably correct. Have rebased a couple times before now but will figure out why this is here.